spec/controllers/hyrax/admin/collection_type_participants_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+RSpec.describe Hyrax::Admin::CollectionTypeParticipantsController, type: :controller do
+  context 'anonymous user' do
+    describe '#create' do
+      it 'returns http redirect' do
+        post :create
+        expect(response).to have_http_status(:redirect)
+      end
+    end
+
+    describe '#destroy' do
+      it 'returns http redirect' do
+        delete :destroy, params: { id: :id }
+        expect(response).to have_http_status(:redirect)
```

Comment:
This spec needs to assert that the `#destroy` action failed. Both success and failure results in a redirect. What is the differentiation?